matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/megolm/MXMegolmDecryption.kt

```diff
@@ -327,9 +327,9 @@ internal class MXMegolmDecryption(private val userId: String,
      * @param senderKey the session sender key
      * @param sessionId the session id
      */
-    override fun onNewSession(senderKey: String, sessionId: String) {
+    fun onNewSession(roomId: String?, senderKey: String, sessionId: String) {
```

Comment:
Please, can you update the documentation about this new param? Also, it would be great if we could have some documentation about the NewSessionListener interface (which would probably be the same as this method) :)

Reply:
Thanks for the explanation, the link between the megolm session and the room was unclear to me!

Reply:
Thanks for the docs update! It would be perfect if you could also add the new param in the doc of that function to be up to date with your changes

Reply:
ah my bad, I thought you meant the interface, will update this method as well :+1:

Reply:
My comment was for both, the interface and the current method which already had documentation :)

Reply:
updated both and provided context on why the value may be null https://github.com/vector-im/element-android/commit/57bf044df0092e9bd9306d17264cfe49e16e4495